app/Exceptions/Paddle/SubscriptionException.php

```
@@ -21,6 +38,27 @@ public function errorCode(): string
+    /** @return array<string, string|null> */
+    public function context(): array
+    {
+        return array_filter([
+            'action' => $this->action,
+            'plan' => $this->plan,
+            'current_state' => $this->currentState,
+        ], fn (?string $value): bool => $value !== null);
+    }
+
+    #[Override]
+    public function meta(Request $request): array
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$request'. (undefined)

(UnusedFormalParameter)